.github/workflows/pr.yaml

```
@@ -22,6 +22,14 @@ jobs:
         run: cargo jjs-check --no-default --rustfmt
       - name: run shellcheck
         run: cargo jjs-check --no-default --shellcheck
+  misspell-fixer:
+    name: misspell-fixer
+    runs-on: "ubuntu-18.04"
+    steps:
+      - uses: actions/checkout@v2
+      - uses: sobolevn/misspell-fixer-action@master
```

```suggestion
      - uses: sobolevn/misspell-fixer-action@v0.1.0
```
or something similar. See https://github.com/sobolevn/misspell-fixer-action/releases